### PR TITLE
Migrate to aiohttp, async

### PR DESCRIPTION
**Pull request by bjw-s**
Required for https://github.com/cgarwood/homeassistant-fullykiosk/pull/23

Also fixes the `setAudioVolume` command, as it was passing `volume` instead of `level`

### CODE REVIEW

File: fullykiosk/__init__.py [hunk 1]
```diff
@@ -1,95 +1,123 @@
+import aiohttp
 import json
 import logging
 import requests
```
Comment (cgarwood) [inline fullykiosk/__init__.py]: leftover import?

**Review by cgarwood — APPROVED** [review]
Looks great! Just 1 super-small comment.

File: fullykiosk/__init__.py (hunk 1 above)
Comment (bjw-s) [reply fullykiosk/__init__.py]: It's not 100% leftover as I reference it on line 95, but the strict type definition is not completely necessary of course. I can remove it if you think that's cleaner.

Comment (cgarwood) [reply fullykiosk/__init__.py]: was referring to the `import requests` line, GitHub didn't do a good job referencing the correct line 😄

**Comment by bjw-s** [comment]
This one should be done I think :) I added some additional error handling so the user will receive an error message when a service call goes wrong (e.g. trying to reboot a device without the correct root permissions)

File: fullykiosk/__init__.py (hunk 1 above)
Comment (bjw-s) [reply fullykiosk/__init__.py]: Doh 😆 fixed...

**Comment by cgarwood** [comment]
Awesome!

**Pull request merged**